My-Portfolio.html

```diff
@@ -0,0 +1,41 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div>
+      <header id="header">
+        <div class="flex-container">
+          <nav id="nav-bar">
+            <div id="logo-placeholder">
+              <h2>John Doe</h2>
+            </div>
+            <div id="normal-button">
+              <img id="menu-button" src="icons/Icon - Menu.svg" alt="menu-button"/>
+            </div>
+          </nav>
+        </div>
+      </header>  
+      <div>
+        <section id="headline">
+          <p id="primary-text">Hey There. I'm Eduardo</p>
```

Comment:
- [ ] [REQUIRED] This primary text should be wrapped with an `h1` element instead of `p` tag because of SEO reasons. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.